xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -231,7 +231,7 @@ smtp_server=Server
 smtp_port=Port
 smtp_server_username=SMTP Server Username (optional)
 smtp_server_password=SMTP Server Password (optional)
-javamail_extra_props=Additional JavaMail properties
+javamail_extra_props=Additional Java Mail properties
```

Comment:
This is not exactly English, "JavaMail" is a Java API which is named like this (yes no white space).